tests/manual-test-cases/Group19-ROBO/19-3-ROBO-VM-Placement.md

```diff
@@ -0,0 +1,36 @@
+Test 19-3 - ROBO - VM Placement
```

Comment:
We should be able to control host resource consumption and test with that in mind.  For example deploy  `progrium/stress` containerVMs that will consume resources in a predictable manner -- once the hosts are running at a known level of consumption then deploy test containerVMs (ubuntu, busybox, etc) and ensure they are placed on the host that we _know_ has the necessary free resources.

Additionally, we need a negative test -- i.e. all hosts are consumed, so deployment fails.

Reply:
Good call, I'll add these details.

Reply:
Fixed.